extensions/kogito/deployment/src/main/java/io/quarkus/kogito/deployment/KogitoAssetsProcessor.java

```diff
@@ -108,6 +109,44 @@ public void generatePersistenceInfo(ArchiveRootBuildItem root,
         }
     }
 
+    @BuildStep
+    public List<ReflectiveIgnoreWarningBuildItem> reflectiveDMNREST() {
+        final DotName ORG = DotName.createComponentized(null, "org");
+        final DotName ORG_KIE = DotName.createComponentized(ORG, "kie");
```

Comment:
I obviously can't  vouch for the correctness of these, but I trust you tried them and they do what you expect them to :)

Reply:
indeed tested ! ;) thanks 👍